package.json

```
@@ -14,12 +15,11 @@
     },
     "description": "",
     "scripts": {
-        "build": "lerna run build",
+        "build": "yarn workspaces foreach run build",
```

```suggestion
        "build": "yarn workspaces foreach -t run build-package",
```
https://yarnpkg.com/cli/workspaces/foreach#options